src/ui/MessageDialog.vala

```diff
@@ -33,7 +33,8 @@ namespace Ev3devKit.Ui {
     public class MessageDialog : Ev3devKit.Ui.Dialog {
         Scroll vscroll;
         Label title_label;
-        Label message_label;
+
+        public Label message_label { get; construct; }
```

Comment:
With regard to your comments on ev3dev/ev3dev#394, what if we just add a constructor with a Widget for a parameter. Instead of using the label created by the current constructor, the widget will be added to the vertical scroll. Then you can add whatever you want to the message if you have a special use case.